proto/klyraprotocol/yields/genesis.proto

```
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package klyraprotocol.yields;
+
+option go_package = "github.com/StreamFinance-Protocol/stream-chain/protocol/x/yield/types";
+
+ message GenesisState {}
```

Buf (1.47.2)

Message "GenesisState" should have a non-empty comment for documentation.

(COMMENT_MESSAGE)